src/main/kotlin/org/web3j/container/ServiceBuilder.kt

```diff
@@ -68,7 +69,12 @@ class ServiceBuilder {
             NodeType.BESU -> BesuContainer(version, resourceFiles, hostFiles, genesisPath)
             NodeType.GETH -> GethContainer(version, resourceFiles, hostFiles, genesisPath)
             NodeType.PARITY -> throw RuntimeException("Container Type Not Supported: $type")
-            NodeType.EMBEDDED -> EmbeddedService(Configuration(Address(selfAddress), 10), PassthroughTracer())
+            NodeType.EMBEDDED -> {
+                if (genesisPath == "dev")
+                    EmbeddedService(Configuration(Address(selfAddress), 10, null), PassthroughTracer())
```

Comment:
No need to pass in null here because of default value.

Reply:
true, i will change it right away